lua/lazyvim/plugins/extras/lang/ansible.lua

```diff
@@ -0,0 +1,36 @@
+return {
+  {
+    "nvim-treesitter/nvim-treesitter",
+    opts = function(_, opts)
+      if type(opts.ensure_installed) == "table" then
+        vim.list_extend(opts.ensure_installed, { "ninja", "yaml" })
```

Comment:
`ninja` != `jinja2`. There aren't any queries and grammars support for `jinja2` in `nvim-treesitter` yet.